.babelrc

```diff
@@ -12,5 +12,17 @@
         "regenerator": false
       }
     ]
-  ]
+  ],
+  "env": {
+    "test": {
+      "presets": [
+        [
+          "@babel/preset-env",
+          {
+            "targets": "current node"
```

Comment:
This fixes the tests. Here is the explanation of the test failure:

1. jest is using babel to compile the code
2. babel was compiling the code to es5 / ie11 format when running the tests
3. To help with compilation, babel was calling `require('@babel/runtime/helpers/<name-of-helper>')`
4. Babel runtime was in ES module format instead of CJS. Jest does not yet support ES modules.

The fix tells babel to use "current node" as a target, and the current node has many more features than IE11. This avoids the need to use the babel runtime helpers during the tests, which avoids the ES module issue.